<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLicensorWindow</class>
 <widget class="QMainWindow" name="QLicensorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QLicensorWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="Directory_TreeView"/>
       <widget class="QGroupBox" name="ChooseLicense_GroupBox">
        <property name="title">
         <string>Choose license to apply</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="AddLICENSE_CheckBox">
           <property name="text">
            <string>Add LICENSE file to project directory</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="ApplyLicense_PushButton">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="ProjectYear_LineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="ProjectDescription_LineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ProjectName_LineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ProjectYear_Label">
           <property name="text">
            <string>Year</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="ProjectAuthor_LineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="ProjectAuthor_Label">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="ProjectName_Label">
           <property name="text">
            <string>Project name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QListWidget" name="ChooseLicense_ListWidget">
           <item>
            <property name="text">
             <string>GNU Public License (GPL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GNU Lesser Public License (LGPL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GNU Affero Public License (AGPL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GNU Free Document License (FDL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BSD 2-Clause License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BSD 3-Clause License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BSD 4-Clause License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Apache License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mozilla Public License (MPL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MIT License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Eclipse Public License</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Public Domain (Unlicense)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ProjectDescription_Label">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="ApplyToSubdirectories_CheckBox">
           <property name="text">
            <string>Apply to files in subdirectories</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="LicenseInformations_groupBox">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>License informations</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="LicenseName_Label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>License Name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="LicenseLogo_Label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/Resources/Bitmap/GPLv3.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>37</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="ShowLicense_PushButton">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Show full license</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="EditHeader_PushButton">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Edit Header</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>86</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>101</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="AuthorLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Author&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="Author_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LatestVersionLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Latest version&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="LatestVersion_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="PublisherLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Publisher&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="Publisher_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="PublishedLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Published&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="Published_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="DFSGCompatibleLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;DFSG compatible&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="DFSGCompatible_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="FSFApprovedLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;FSF approved&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="FSFApproved_Label">
            <property name="text">
             <string>&lt;font color=&quot;green&quot;&gt;Yes&lt;/font&gt;</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="OSIApprovedLabel_label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;OSI approved&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="OSIApproved_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="GPLCompatibleLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;GPL compatible&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="GPLCompatible_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="CopyleftLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Copyleft&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="Copyleft_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="LinkingLabel_Label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Linking&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="Linking_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="WebsiteLable_Lable">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;b&gt;Website&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="Website_Label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionLoadDirectory"/>
    <addaction name="actionRecent_directories"/>
    <addaction name="actionCloseWorkingDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQtFramework"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadDirectory"/>
   <addaction name="actionCloseWorkingDirectory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadDirectory">
   <property name="icon">
    <iconset theme="archive-insert-directory">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Load Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseWorkingDirectory">
   <property name="text">
    <string>Close Working Directory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionRecent_directories">
   <property name="text">
    <string>Recent directories</string>
   </property>
  </action>
  <action name="actionAboutQtFramework">
   <property name="text">
    <string>About &amp;Qt Framework</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ChooseLicense_ListWidget</tabstop>
  <tabstop>ProjectName_LineEdit</tabstop>
  <tabstop>ProjectDescription_LineEdit</tabstop>
  <tabstop>ProjectAuthor_LineEdit</tabstop>
  <tabstop>ProjectYear_LineEdit</tabstop>
  <tabstop>ApplyToSubdirectories_CheckBox</tabstop>
  <tabstop>AddLICENSE_CheckBox</tabstop>
  <tabstop>ApplyLicense_PushButton</tabstop>
  <tabstop>ShowLicense_PushButton</tabstop>
  <tabstop>EditHeader_PushButton</tabstop>
  <tabstop>Directory_TreeView</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
